pkg/framework/context/context.go

```diff
@@ -7,16 +7,16 @@ SPDX-License-Identifier: Apache-2.0
 package context
 
 import (
-	exchangeService "github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/exchange/service"
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
 	"github.com/hyperledger/aries-framework-go/pkg/framework/aries/api"
 	errors "golang.org/x/xerrors"
 )
 
 // Provider supplies the framework configuration to client objects.
 type Provider struct {
-	outboundTransport transport.OutboundTransport
-	apiHandlers       []api.Handler
+	outboundTransport  transport.OutboundTransport
+	apiHandlers        []api.Handler
+	protocolSvcCreator []api.ProtocolSvcCreator
```

Comment:
Technically a plural? Also I saw similar in another file.

Reply:
done